src/langs/ar.json

```diff
@@ -128,70 +142,105 @@
   "ITEM_SETTINGS_CC_LICENSE_INFORMATIONS": "جميع المحتويات المنشورة على مكتبة چراسپ لا تسمح بالاستخدام التجاري.\n هل تسمح بمشاركة تعديلات تمّت على عملك؟",
   "ITEM_SETTINGS_CC_LICENSE_LABEL": "رخصة المشاع الإبداعي",
-  "ITEM_SETTINGS_CC_LICENSE_MODAL_CONTENT": "يرجى التحقق من أن العنصر الخاص بك يتناسب مع رخصة المشاع الإبداعي، ولا تقم بالتغيير إلى خيار أكثر تقييدًا.",
-  "ITEM_SETTINGS_CC_LICENSE_MODAL_TITLE": "ثبّت تقديمك",
+  "ITEM_SETTINGS_CC_LICENSE_MODAL_CONTENT": "يرجى التحقق من أن العنصر الخاص بك يتناسب مع رخصة CC، ولا تقم بالتغيير إلى خيار أكثر تقييدًا.",
+  "ITEM_SETTINGS_CC_LICENSE_MODAL_TITLE": "المصادقة على التسليم",
   "ITEM_SETTINGS_CC_LICENSE_MORE_INFORMATIONS": "هل تحتاج إلى مزيد من المعلومات حول ترخيص CC؟ انقر هنا!",
   "ITEM_SETTINGS_CC_LICENSE_NONE_LABEL": "لا شَيْء",
   "ITEM_SETTINGS_CC_LICENSE_PREVIEW_TITLE": "معاينة الرمز",
   "ITEM_SETTINGS_CC_LICENSE_SUBMIT_BUTTON": "تقديم",
   "ITEM_SETTINGS_CC_LICENSE_TITLE": "رخصة المشاع الإبداعي",
+  "ITEM_SETTINGS_CO_EDITORS_INFORMATIONS": "عرض المحررين المشاركين بعد النشر",
   "ITEM_SETTINGS_CO_EDITORS_TITLE": "المحرّرين المشاركين",
   "ITEM_SETTINGS_VISIBILITY_CANNOT_EDIT_PARENT_MESSAGE": "لا يمكنك تعديل حالة الظهور لأنه معرّف في أحد العناصر الأصل لهذا العنصر.",
   "ITEM_SETTINGS_VISIBILITY_PRIVATE_INFORMATION": "هذا العنصر خاص. يمكن للأعضاء المصّرح لهم فقط الوصول إليه.",
   "ITEM_SETTINGS_VISIBILITY_PRIVATE_LABEL": "ّخاص",
   "ITEM_SETTINGS_VISIBILITY_PSEUDONYMIZED_LABEL": "اسم مستعار",
-  "ITEM_SETTINGS_VISIBILITY_PSEUDONYMIZED_SCHEMA_PSEUDONYM_AND_PASSWORD_LABEL": "الأُسم المستعار وكلمة المرور",
-  "ITEM_SETTINGS_VISIBILITY_PSEUDONYMIZED_SCHEMA_PSEUDONYM_LABEL": "الأُسم المستعار",
+  "ITEM_SETTINGS_VISIBILITY_PSEUDONYMIZED_SCHEMA_PSEUDONYM_AND_PASSWORD_LABEL": "الاسم المستعار وكلمة المرور",
+  "ITEM_SETTINGS_VISIBILITY_PSEUDONYMIZED_SCHEMA_PSEUDONYM_LABEL": "الاسم المستعار",
   "ITEM_SETTINGS_VISIBILITY_PSEUDONYMIZED_SCHEMA_SELECT_MESSAGE": "يمكن الوصول إلى هذا العنصر إذا قدم الزائر ملفًا",
   "ITEM_SETTINGS_VISIBILITY_PUBLIC_INFORMATIONS": "هذا العنصر عام. يمكن لأي شخص الوصول إليه.\n ملاحظة: لن يتم نشر العناصر تلقائيًا إذا قمت بتغيير حالة الظهور من عام إلى آخرين.",
   "ITEM_SETTINGS_VISIBILITY_PUBLIC_LABEL": "عام",
   "ITEM_SETTINGS_VISIBILITY_TITLE": "حالة الظّهور",
+  "ITEM_TAGS_PLACEHOLDER": "أضف علامات مثل اللغة الإنجليزية والأحياء والمختبر والنباتات والمدرسة الثانوية",
+  "ITEM_TAGS_CHIP_BUTTON_EDIT": "تعديل العلامات",
   "ITEM_TAGS_TITLE": "العلامات",
+  "ITEM_TAGS_LABEL": "تعديل العلامات",
+  "ITEM_TAGS_MISSING_WARNING": "أضف علامة واحدة على الأقل لتوضيح الأشخاص الذين يمكنهم استخدام منشورك بشكل أفضل",
   "ITEMS_GRID_ITEMS_PER_PAGE_TITLE": "العناصر لكل صفحة",
-  "ITEMS_TABLE_ACTIONS_HEADER": "أجراءات",
+  "ITEMS_TABLE_ACTIONS_HEADER": "اجراءات",
   "ITEMS_TABLE_CREATOR_HEADER": "المُنشئ",
   "ITEMS_TABLE_DRAG_DEFAULT_MESSAGE": "نقل عنصر واحد",
   "ITEMS_TABLE_EMPTY_MESSAGE": "لا توجد عناصر",
```

Comment:
@LinaYahya There is currently a conflict here. The translation for this key has been changed in the source english version from `No items` to `This folder does not contain any items`. Either the current translation in arabic corresponds to this use-case or it needs to be adapted to fit the slight variation introduced in the english version. Thank you in advance.